src/main/java/eu/europa/ted/eforms/sdk/SdkAnalyzerSymbolResolver.java

```diff
@@ -0,0 +1,26 @@
+package eu.europa.ted.eforms.sdk;
+
+import java.nio.file.Path;
+import eu.europa.ted.eforms.sdk.SdkConstants.SdkResource;
+import eu.europa.ted.eforms.sdk.repository.SdkCodelistRepository;
+import eu.europa.ted.eforms.sdk.repository.SdkFieldRepository;
+import eu.europa.ted.eforms.sdk.repository.SdkNodeRepository;
+
+public class SdkAnalyzerSymbolResolver extends SdkSymbolResolver {
+  public SdkAnalyzerSymbolResolver(String sdkVersion, Path sdkRootPath)
+      throws InstantiationException {
+    super(sdkVersion, sdkRootPath);
+  }
+
+  @Override
+  protected void loadMapData(final String sdkVersion, final Path sdkRootPath)
+      throws InstantiationException {
+    Path jsonPath = Path.of(sdkRootPath.toString(), SdkResource.FIELDS_JSON.getPath().toString());
```

Comment:
`sdkRootPath.resolve(SdkResource.FIELDS_JSON.getPath())` would avoid using toString() twice.
But it is also OK like this.